lang.ms.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*------------------
+Language: Bahasa Melayu
+------------------*/
+
+$lang = array();
+
+$lang['BEEIMG'] = 'BeeIMG';
+
+// Titles
+$lang['TITLE_HOME'] = 'Hosting dan Kongsi Gambar Percuma';
+$lang['TITLE_UPLOAD'] = 'Muat Naik Gambar';
+$lang['TITLE_RECENT'] = 'Gambar Terbaru';
+$lang['TITLE_STATS'] = 'Statistik';
+$lang['TITLE_TOP'] = '9 Gambar Teratas';
+$lang['TITLE_ABOUT'] = 'Mengenai Kami';
+$lang['TITLE_CONTACT'] = 'Hubungi';
+$lang['TITLE_TOS'] = 'Terma Perkhidmatan';
+$lang['TITLE_REGISTER'] = 'Daftar';
+$lang['TITLE_LOGIN'] = 'Log Masuk';
+$lang['TITLE_DASHBOARD'] = 'Dashboard';
+
+// Menu
+$lang['MENU_HOME'] = 'Laman Utama';
+$lang['MENU_HOME_TITLE'] = 'Laman Utama Kami';
+$lang['MENU_UPLOAD'] = 'Muat Naik';
+$lang['MENU_UPLOAD_TITLE'] = 'Muat Naik Gambar';
+$lang['MENU_RECENT'] = 'Gambar Terbaru';
+$lang['MENU_RECENT_TITLE'] = 'Gambar Muat Naik Terbaru';
+$lang['MENU_STATS'] = 'Statistik';
+$lang['MENU_STATS_TITLE'] = 'Lihat Statistik Kami';
+$lang['MENU_TOP'] = '9 Teratas';
+$lang['MENU_TOP_TITLE'] = 'Lihat 9 Gambar Teratas';
+$lang['MENU_FOLLOW'] = 'Ikuti';
+
+// Upload Carousel
+$lang['UPLOAD_CAR_LP'] = 'Laman Utama';
+$lang['UPLOAD_CAR_1'] = 'Tiada IP direkodkan semasa muat naik.';
+$lang['UPLOAD_CAR_2'] = 'Kami mengawal muat naik anda.';
+$lang['UPLOAD_CAR_3'] = 'Tiada iklan dibenarkan.';
+
+// Footer
+$lang['FOOTER_CONTACT'] = 'hubungi';
+$lang['FOOTER_API'] = 'api';
```

Comment:
Leave it as API not api . Because api when translated from Malay to English is 'fire'